mem/mem_linux.go

```diff
@@ -156,3 +159,39 @@ func SwapMemoryWithContext(ctx context.Context) (*SwapMemoryStat, error) {
 	}
 	return ret, nil
 }
+
+func calcuateAvailVmem(ret *VirtualMemoryStat) uint64 {
```

Comment:
Could you add psutil PR link, https://github.com/giampaolo/psutil/pull/890 which is merged PR as comment?

Reply:
👍 i will